tests/releasenotes/notes/floyd-warshall-fix-507ddeb9b792c89f.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This file is in the wrong directory, it should be `releasenotes/notes/` but this is in in `tests/releasenotes/notes` which means it won't show up in the release notes in the documentation.

Reply:
Nice catch! Apparently I did the same thing for another release note too. It should be fixed now